schema.sql

```diff
@@ -8,3 +8,5 @@ CREATE TABLE animals (
     neutered BOOLEAN,
     weight_kg DECIMAL(10, 2)
 );
+
+ALTER TABLE animals ADD COLUMN species VARCHAR(255);
```

Comment:
- [ ] Here the expected new tables and table modification have not been done. Please check the project requirements.

```sql
   CREATE TABLE owners (
   id int NOT NULL IDENTITY(1,1),
   full_name varchar(255),
   age int,
   PRIMARY KEY (id)
   );

-- others here
```

- [ ] I strongly suggest that the query screenshots be arranged in the pull request description space to separate each screenshot like this 👇🏾 

- Who owns the most animals?

![image](https://user-images.githubusercontent.com/84156714/220784277-eb439540-0600-45b5-acf4-d28877c7f028.png)

- List all Digimon owned by Jennifer Orwell.

![image](https://user-images.githubusercontent.com/84156714/220784429-ae1ff965-913a-4edf-a4a6-a4d06b9504b2.png)

#### This will make it easy for the reviewer to quickly understand your queries.